week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -55,12 +90,27 @@ let answer;
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
+function checkAnswer(randomArray) {
+  console.log('second function', randomAnswear)
+  if (randomArray === answears[0]) {
```

Comment:
This is the right idea, but unfortunately one thing that we learnt in week 2 is that you can't compare arrays with `===`, so this always returns `very negative` :( This tricked our tests, though, because it's really hard to write tests which test random behaviour (a problem we see in professional software engineering too!!)

There are a few ways that we could do this kind of check, and we're going to learn about them more in week 3 (where we'll learn more about arrays, and I think objects too) :)

An example  is that we could use a loop to go through each array, and look for `randomAnswear` in them. Something like:
```javascript
for (let i = 0; i < 5; ++i) {
  if (answears[i].includes(randomAnswear)) {
    if (i === 0) {
      return "Very positive";
    } else if (i === 1) {
      return "positive";
    }
    ...
}
```